src/crimson/osd/heartbeat.cc

```diff
@@ -375,7 +399,6 @@ seastar::future<> Heartbeat::send_failures()
                                                   osdmap->get_addrs(osd)});
       return monc.send_message(failure_report);
     }).then([this] {
-      failure_queue.clear();
```

Comment:
please remove this continuation along with `failure_queue.clear()`, we don't need it anymore.

Reply:
Um.. it seems this is not my most recent update..